caches/path-mapped/src/main/java/org/commonjava/maven/galley/cache/pathmapped/util/CassandraPathDBUtils.java

```diff
@@ -0,0 +1,49 @@
+package org.commonjava.maven.galley.cache.pathmapped.util;
+
+public class CassandraPathDBUtils
+{
+    public static final String PROP_CASSANDRA_HOST = "cassandra_host";
+
+    public static final String PROP_CASSANDRA_PORT = "cassandra_port";
+
+    public static final String PROP_CASSANDRA_KEYSPACE = "cassandra_keyspace";
+
+    public static String getSchema_CREATE_KEYSPACE( String keyspace )
```

Comment:
Actually from java specification, it is not good practice to use "_" in method name. Maybe getSchemaCreateKeyspace is better.

Reply:
Ok.

Reply:
Actually I don't know if I need the param... I will test it with keyspace specified somewhere else. If ok, I will roll back to simple constants.